spec/feature/logout_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.feature "Logout" do
+  before do
+    reset_application!
+  end
+
+  scenario "Logging out redirects to home page" do
```

Comment:
It's unclear what this is testing, shouldn't we test that the page has "ANNE MERICA (283)" on the screen before logging out, and does not have "ANNE MERICA (283)" after?